sdk/storage/Azure.Storage.Files/src/Generated/FileRestClient.cs

```diff
@@ -7435,139 +7338,105 @@ public partial class ShareCreatePermissionResult
     public static partial class FilesModelFactory
     {
         /// <summary>
-        /// Creates a new ShareCreatePermissionResult instance for mocking.
+        /// Creates a new RawStorageDirectoryInfo instance for mocking.
         /// </summary>
-        public static ShareCreatePermissionResult ShareCreatePermissionResult(
-            string filePermissonKey)
+        public static RawStorageDirectoryInfo RawStorageDirectoryInfo(
+            Azure.Core.Http.ETag eTag,
+            System.DateTimeOffset lastModified,
+            string filePermissionKey,
+            string fileAttributes,
+            System.DateTimeOffset fileCreationTime,
+            System.DateTimeOffset fileLastWriteTime,
+            System.DateTimeOffset fileChangeTime,
+            string fileId,
+            string fileParentId)
         {
-            var _model = new ShareCreatePermissionResult();
-            _model.FilePermissonKey = filePermissonKey;
+            var _model = new RawStorageDirectoryInfo();
+            _model.ETag = eTag;
+            _model.LastModified = lastModified;
+            _model.FilePermissionKey = filePermissionKey;
+            _model.FileAttributes = fileAttributes;
+            _model.FileCreationTime = fileCreationTime;
+            _model.FileLastWriteTime = fileLastWriteTime;
+            _model.FileChangeTime = fileChangeTime;
+            _model.FileId = fileId;
+            _model.FileParentId = fileParentId;
             return _model;
         }
     }
 }
-#endregion class ShareCreatePermissionResult
+#endregion class RawStorageDirectoryInfo
 
-#region class ShareInfo
+#region class RawStorageDirectoryProperties
```

Comment:
Open to renaming this.  Same with all the generated Raw* classes.

Reply:
I think `Raw` is a fine prefix for these.